spec/controllers/platform/api/v1/accounts_controller_spec.rb

```diff
@@ -144,14 +140,19 @@
 
         patch "/platform/api/v1/accounts/#{account.id}", params: {
           name: 'Test Account',
-          enabled_features: %w[feature_ip_lookup feature_help_center],
+          features: {
```

Comment:
@smartdev58 when specifying  { `feature_name`:  false } , will it disable the feature if it was already enabled ? 
If so, can you update this test case to ensure that?

Reply:
> @smartdev58 when specifying { `feature_name`: false } , will it disable the feature if it was already enabled ? If so, can you update this test case to ensure that?

@sojan-official I have already done the update you mentioned for the patch. line: 146: `channel_facebook: false`